src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,33 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const initalState = [];
+const booksReducer = (state = initalState, action) => {
+  switch (action.type) {
+    case 'ADD_BOOK':
+      return [
+        ...state,
+        {
+          id: action.id,
+          title: action.payload.title,
+          author: action.payload.author,
+        },
+      ];
+
+    case 'REMOVE_BOOKS':
+      return state.filter((book) => book.id !== action.id);
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Nice work creating the reducer :clap: However it is requested in the project requirement to use `createSlice` API to define the state for book. It is the standard approach for writing Redux logic. You can read more about `createSlice` [here](https://redux-toolkit.js.org/api/createslice). Therefore I suggest you use it to follow the project requirement.
![Screenshot from 2023-08-21 16-05-59](https://github.com/katarighe/bookstore/assets/91142432/52ae4449-d985-400e-81f0-e1f1b3fb183f)